### PR TITLE
Scib integration

### PR DESCRIPTION
**Pull request by andrewram4287**
Using the format from Theis Lab (SCIB)

### CODE REVIEW

**Comment by andrewram4287** [comment]
@aarmey Is this what you were thinking of when integrating our method with their format?

**Comment by aarmey** [comment]
@andrewram4287 can you move over the test code I wrote and try that? That should give you a very good idea of whether your code is working.

File: sccp/imports/scRNA.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def import_thompson_drug() -> anndata.AnnData:
     return data
 
 
-def ThompsonXA_SCGenes(offset: float = 1.0) -> anndata.AnnData:
+def ThompsonXA_SCGenes(annData = False, offset: float = 1.0) -> anndata.AnnData:
```
Comment (aarmey) [inline sccp/imports/scRNA.py:58]: Why are you messing with this function? I wrote a test with data from the Theis lab.

File: sccp/figures/figure0.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                    f"Deleting duplicated keys `{list(df.columns[dup_mask].unique())}` from `adata.{attr}`."
+                )
+                setattr(_adata, attr, df.loc[:, ~dup_mask])
+
+    return anndata.AnnData.concatenate(*adata_list, **kwargs)
+    
+    
+def aDataCorrected_Pf2(rank, annData=True):
+    """Imports data as both tensor and AnnData. Saves projections in AnnData for SCIB/Theis Lab"""
+    aData, X = ThompsonXA_SCGenes(annData=annData, offset=1.0)
```
Comment (aarmey) [inline sccp/figures/figure0.py:88]: Again, you should be using the test I wrote, not assuming you know what the file format is.

File: sccp/figures/figure0.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    subplotLabel,
+    getSetup,
+    flattenData
+)
+from ..parafac2 import parafac2_nd
+import numpy as np
+from ..imports.scRNA import ThompsonXA_SCGenes
+import numpy as np
+
+def makeFigure():
```
Comment (aarmey) [inline sccp/figures/figure0.py:15]: No need for this function if the interface function is called through a test.

**Pull request closed without merging**